ethereum/contracts/TablelandRigs.sol

```diff
@@ -380,6 +641,7 @@ contract TablelandRigs is
 
     /**
      * @dev See {ERC721A-_beforeTokenTransfers}.
+     * Also, adds logic to block transfers while a Rig is being piloted.
```

Comment:
It might be useful to also add an `_afterTokenTransfer` table insert that helps track Rig ownership. I.e., a simple `rigs_ownership_index` table with `rig_id` and `owner`'s address.

Cross posting some comments about it: [see here](https://github.com/tablelandnetwork/rigs/commit/1d97b32ba315758d3ddc2a8a22a9bf4ece9bbf1b#r86312956)

Reply:
dig it